web-server/v0.4/src/models/global.js

```diff
@@ -17,6 +20,37 @@ export default {
         payload: response,
       });
     },
+    *fetchMonthIndices({ payload }, { call, put }) {
+      let response = yield call(queryMonthIndices, payload);
+
+      let indices = [];
+      response.map(index => {
+        if (index.index.includes('dsa.pbench.run')) {
```

Comment:
Use of `dsa.pbench` here should probably be replaced with the prefix value from the configuration file.

Reply:
@portante The global redux model now takes advantage of the datastoreConfig file to build the prefix for reference when filtering month indices.